impeller/scene/geometry.h

```diff
@@ -32,5 +42,15 @@ class CuboidGeometry final : public Geometry {
   Vector3 size_;
 };
 
+class VertexBufferGeometry final : public Geometry {
+ public:
+  void SetVertexBuffer(VertexBuffer vertex_buffer);
+
+  VertexBuffer GetVertexBuffer(Allocator& allocator) const override;
```

Comment:
Nit: For later if needed. Let's stick to the style guide and note the superclass that defines this overriden method.